src/lib/wallet.test.js

```diff
@@ -0,0 +1,256 @@
+import '../../src/installSesLockdown';
+import { test, expect, describe, vi } from 'vitest';
+
+vi.stubGlobal('window', {
+  location: { search: '' },
+  alert: vi.fn(),
+  keplr: {
+    experimentalSuggestChain: vi.fn(),
+    enable: vi.fn(),
+    getKey: () => ({ isNanoLedger: true }),
+    getOfflineSignerOnlyAmino: () => ({
+      getAccounts: () => [{ address: 1 }],
+    }),
+  },
+});
+
+import { inferInvitationStatus } from './wallet';
+import { LoadStatus } from './rpc';
+
+const createMock = ({
```

Comment:
I'd rename this to `createMockWallet` or something similar, and instead of `mock`s below, `mockWallet` or such.